rep-I0001.rst

```diff
@@ -61,18 +61,18 @@ The industrial robot controller motion interface shall meet the following requir
  * The controller joint group structure shall be statically defined at node start up (i.e. in a yaml file read on node construction).  Motion groups are predefined (statically) by most controllers.
  
 Design Assumptions
-=========
+==================
 The following assumptions are inherent in the client/server design:
- * All joints (regardless of group) shall be uniquely named.
- 
+ * All joints (within a common namespace) shall be uniquely named.
  
 Simple Message Structures
-=========
+=========================
 The simple message package [#simp_msg]_ provides libraries for communicating with industrial robot controllers.  This includes connection handling libraries and message packing/unpacking capabilities.  The default robot connection is TCP/IP, although any method of transferring byte data is easily supported.  While not required, traditional message structures have been statically defined (i.e. fixed arrays).  This is because robot controllers cannot dynamically allocate memory.  If dynamic messages are used, controller side servers should utilize fixed size data that comply with some physical limitation (i.e. the controller can only handle ten axes in a single group) and then handle the error cases when the simple message exceeds that amount.  By creating dynamic simple messages for motion and status, multiple arm control and monitoring can be achieved.
 
+In addition to dynamic array-sizing, several of the new messages also allow for populating only a subset of the supported message fields.  This allows for the messages to support a wide range of potential data, while also allowing robot controllers and ROS control algorithms to only support a small subset of those capabilities.  The new messages contain a ``valid_fields`` item, that signals which of the potential data fields are being used.  The messages are packed, to minimize bandwidth, so that only the specified fields are acutally transmitted in the message datastream.  The ordering of data fields is still maintained, even when some fields are skipped.
 
 Dynamic Joint Point
----------
+-------------------
 The dynamic joint point is meant to mimic the ROS JointTrajectory message structure [#traj_msg]_.  A one-to-one mapping of the joints included in the ROS message to the simple message shall be created.  By encapsulating the entire trajectory in a single message, synchronized motion is possible.::
```

Comment:
> By encapsulating the entire trajectory in a single message, synchronized motion is possible.

Isn't synchronised motion only dependent on the trajectory point containing info for all groups that are to be part of the motion? A `Dynamic Joint Point` is still just one point, right?